src/coreComponents/dataRepository/GroupContext.cpp

```diff
@@ -34,22 +34,30 @@ GroupContext::GroupContext( Group & group ):
 
 string GroupContext::toString() const
 {
-  string path;
-  bool foundNearestLine = false;
-  for( Group const * parentGroup = &m_group; parentGroup->hasParent(); parentGroup = &parentGroup->getParent() )
+  std::vector< string > parents;
+  Group const * parentGroup = &m_group;
+  // add all parent names in a path string, until we get some input file info to show
+  bool foundNearestLineInfo = false;
```

Comment:
So do you still need this `bool` here? It seems like it's never modified?
Surely you need to be doing something with it... Is this code doing what you want?

Personnaly, I would have added the extra gathering of `std::vector< ToStringInfo >` before converting to `strings`.

Reply:
I also pushed another version of `GroupContext::toString()` that starts by loading all `ToStringInfo`.
It may be better in term of readability.